src/quo/components/avatars/user_avatar/view.cljs

```diff
@@ -1,11 +1,13 @@
 (ns quo.components.avatars.user-avatar.view
   (:require
+    [quo.components.avatars.user-avatar.schema :as component-schema]
     [quo.components.avatars.user-avatar.style :as style]
     [quo.components.common.no-flicker-image :as no-flicker-image]
     [quo.components.markdown.text :as text]
     [quo.theme]
     [react-native.core :as rn]
     [react-native.fast-image :as fast-image]
+    [schema.core :as schema]
     utils.string))
 
 (defn initials-avatar
```

Comment:
I think we can remove the docstring with the addition of the schema

Reply:
yes, that is something we should check for too. We should just go and nuke majority of doc strings unless there is specific notes/comments in there 👍